include/aws/io/retry_strategy.h

```diff
@@ -112,6 +112,8 @@ struct aws_exponential_backoff_retry_options {
     size_t max_retries;
     /** Scaling factor to add for the backoff. Default is 25ms */
     uint32_t backoff_scale_factor_ms;
+    /** Maximum retry backoff in seconds. Default is 20 seconds */
+    uint32_t maximum_backoff_s;
```

Comment:
trivial: we don't usually abbreviate seconds as "s"

searching public `aws/**/*.h` I found 4 each of variables ending in `_sec` or `_secs`

Reply:
debatable: it's a bit weird to mix different units of time (ms and sec). Worth sticking with ms just so it doesn't confuse people?

Reply:
I have changed it to `secs`. I think it's fine to have different units, as we mention them in the name. Updating it to `ms` leads to the question of whether to allow 1ms or have a minimum of 1000ms.